src/features/ministry/report/form_S4/submit_report/useSubmitReport.tsx

```
@@ -104,39 +109,32 @@
       }
-    } while (!valid);
 
-    if (currentMonth === month) {
       month = String(+month + 1).padStart(2, '0');
 
       if (+month === 13) {
         month = '01';
         year = String(+year + 1);
       }
-    }
+    } while (true);
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)